src/gpodder/model.py

```diff
@@ -462,6 +465,19 @@ def update_from_dict(self, episode_dict):
             if k in episode_dict:
                 setattr(self, k, episode_dict[k])
 
+    @property
+    def art_file(self):
+        if self.episode_art_url != None and self.episode_art_url != '':
+            filename = self.guid
```

Comment:
My mistake, I didn't look at the contents of the field :man_facepalming:

Reply:
However I do want to figure out a title to use for such episodes that fail to get a filename through their URL since they do have artwork that needs to be saved.